cli/Squidex.CLI/Squidex.CLI.Tests/MapJsonFileToSquidexTests.cs

```diff
@@ -0,0 +1,307 @@
+﻿// ==========================================================================
+//  Squidex Headless CMS
+// ==========================================================================
+//  Copyright (c) Squidex UG (haftungsbeschraenkt)
+//  All rights reserved. Licensed under the MIT license.
+// ==========================================================================
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Squidex.CLI.Commands;
+using Squidex.CLI.Commands.Implementation;
+using Squidex.ClientLibrary;
+using Xunit;
+
+namespace Squidex.CLI.Tests
+{
+    public class MapJsonFileToSquidexTests
+    {
+        private readonly JsonTextReader jsonReader;
+
+        public MapJsonFileToSquidexTests()
+        {
+            var json = new StringBuilder();
+
+            using (var stringWriter = new StringWriter(json))
```

Comment:
Can you create an anonymous object here and serialize that to json.

Like 

```
var json = JsonConvert.SerializeObject(new {
   text = "..."
});

it is hard to understand what you create here.

Reply:
Good idea, I will change it


> Can you create an anonymous object here and serialize that to json.
> 
> Like
> 
> ```
> var json = JsonConvert.SerializeObject(new {
>    text = "..."
> });
> 
> it is hard to understand what you create here.
> ```